src/main/kotlin/com/safehill/kclient/models/interactions/InteractionAnchor.kt

```diff
@@ -0,0 +1,11 @@
+package com.safehill.kclient.models.interactions
+
+import kotlinx.serialization.SerialName
+
+enum class InteractionAnchor(val serverKey: String) {
```

Comment:
What is serverKey for?

Reply:
Also why is this part of this PR?

Reply:
serverKey is the value that is sent by the server for interaction type in notification.

The notification in Android was being worked on side by side. The register device PR was an aftermath of the notification work on the Android side. The InteractionAnchor was therefore required to distinguish the type of InteractionAnchor when parsing notifications.

Reply:
So what do you suggest is the next step here?

Reply:
I am not sure what you mean by next step.. The InteractionAnchor is used in the Android side for classifying the notification type.

Reply:
> serverKey is the value that is sent by the server for interaction type in notification.
Is this a temporary thing? I guess I don't fully understand the need

Reply:
If you look [in this PR](https://github.com/Safehill/Snoog-Android/blob/217785330fee499646601d2a45310d8e94639889/app/src/main/kotlin/com/safehill/snoog/notifications/notification_message/NotificationMessageHandler.kt), "anchor" key is being converted to InteractionAnchor to distinguish group or thread.

Reply:
That makes sense. What doesn't' make sense to me is the `serverKey`. Perhaps I'm missing something?

Reply:
Maybe the variable naming is throwing you a bit off. 
We receive a value for "anchor" key in the notification. That value is associated with the InteractionAnchor.
The value in "anchor" key is being used to distinguish the type of notification.
In the swift counterpart, SHInteractionAnchor.rawValue is being used to  distinguish notification. SHInteractionAnchor.rawValue's counterpart is the serverKey over here.

Reply:
Got it. Yes the name is throwing me off

Reply:
shall I name it 'value'?